public/app/features/plugins/PluginPage.tsx

```diff
@@ -500,4 +523,71 @@ export function loadPlugin(pluginId: string): Promise<GrafanaPlugin> {
   });
 }
 
+type PluginSignatureDetailsBadgeProps = {
+  signatureType?: PluginSignatureType;
+  signatureOrg?: string;
+};
+
+const PluginSignatureDetailsBadge: React.FC<PluginSignatureDetailsBadgeProps> = ({ signatureType, signatureOrg }) => {
+  const styles = useStyles2(getDetailsBadgeStyles);
+
+  if (!signatureType && !signatureOrg) {
+    return null;
+  }
+
+  const signatureTypeIcon =
+    signatureType === PluginSignatureType.grafana
+      ? 'grafana'
+      : signatureType === PluginSignatureType.commercial || signatureType === PluginSignatureType.community
+      ? 'shield'
+      : 'shield-exclamation';
+
+  const signatureTypeText = signatureType === PluginSignatureType.grafana ? 'Grafana Labs' : capitalize(signatureType);
+
+  return (
+    <>
+      {signatureType && (
+        <Badge
+          color="green"
+          className={styles.badge}
+          text={
+            <>
+              <strong className={styles.strong}>Level:&nbsp;</strong>
```

Comment:
I guess this is a well-thought name, but I personally find Level quite a weird name 😬

Reply:
Yeah, I thought the same plus the code (BE and FE) refers to it as "type".